Packs/IBMResilientSystems/Integrations/IBMResilientSystems/IBMResilientSystems.py

```diff
@@ -115,92 +126,204 @@
     3: 'Individual'
 }
 
+OBJECT_ACTION_TYPE_TO_ID = {
+    'Incident': 0,
+    'Task': 1,
+    'Note': 2,
+    'Milestone': 3,
+    'Artifact': 4,
+    'Attachment': 5,
+    'Email Message': 13
+}
+
+IBM_QRADAR_INCIDENT_FIELDS = {
+    "description": {
+        "xsoar_name": "description",
+        "description": "Description of the incident.",
+    },
+    'incident_type_ids': {
+        'xsoar_name': 'alerttypeid',
+        'description': 'The IDs of the incident types.'
+    },
+    'resolution_id': {
+        'xsoar_name': 'ibmqradarresolution',
+        'description': ''
+    },
+    'resolution_summary': {
+        'xsoar_name': 'ibmqradarresolutionsummary',
+        'description': ''
+    },
+    "owner_id": {
+        "xsoar_name": "",
+        "description": "The principal ID of the incident owner.",
+    },
+    "reporter": {
+        "xsoar_name": "ibmqradarreportername",
+        "description": "Who reported the incident.",
+    },
+    "severity_code": {
+        "xsoar_name": "severity",
+        "description": "The severity of the incident. 4 = Low, 5 = Medium, 6 = High.",
+    },
+    "creator.display_name": {
+        "xsoar_name": "displayname",
+        "description": "The display name of the incident creator.",
+    }
+}
+
+""" CONSTANTS """
+FILE_DOWNLOAD_ERROR_MESSAGE = "<html><head><title>Download error</title></head><body>Download error</body></html>"
+SCRIPT_ENTITIES = "entities"
+DEFAULT_RETURN_LEVEL = "full"
+DEFAULT_RETRIES = 1
+STATUS_NOT_FOUND = 404
```

Comment:
Why not use the HTTPStatus ?
from http import HTTPStatus

Reply:
anyways, this constant was not used.